skycoin/skywire
# Feature | http-proxy on skysocks-client

Did you run `make format && make check`? Yes

Fixes #1696 

 Changes:	
- add `-http` argument to skysocks-client config, as address of http-proxy binding (not available for no on UI, just available on config code or by cli)
- add `--http-proxy` or `-p` flag to `skywire-cli proxy start` to pass http-proxy address to skysocks-client
- update CHANGELOG.md and dmsg@v1.3.17 and prepare for new release

How to test this PR:
- run visor and make transport to itself by dmsg
- run proxy by `skywire-cli proxy start -p :1081 -k <visor-pk>`
- check http proxy on app like Mozilla Firefox network setting
